docs/installation.md

```diff
@@ -88,7 +91,7 @@ Get the latest version from [this page](https://github.com/aquasecurity/trivy/re
 
 ## From source
 
-```sh
+```bash
 $ mkdir -p $GOPATH/src/github.com/aquasecurity
 $ cd $GOPATH/src/github.com/aquasecurity
 $ git clone https://github.com/aquasecurity/trivy
```

Comment:
This will clone and install the latest, not the current version for which I'm reading the docs

Reply:
I didn't want to manage versions in variables. I'm sure I will forget to update them 😄 
Eventually, I found macros supports git information.gg
https://mkdocs-macros-plugin.readthedocs.io/en/latest/git_info/

I've added `{{ git.tag }}`.
https://github.com/aquasecurity/trivy/pull/884/commits/d17f570d494ab2f68fea0b8c53ae0fab82a63135